src/shared/utils/logger.js

```diff
@@ -47,6 +47,7 @@ if (isServerSide()) {
       }),
       headers: {
         'Content-Type': 'application/json',
+        'Authorization': process.env.SERVER_API_KEY
```

Comment:
Node need to wrap `Authorization` into `'`. Also the value should match the format noted above.